_posts/2017/2017-03-28-epages-hackathon-2017.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "ePages Hackathon 2017"
+date: "2017-03-28 06:39:20"
+image: blog-header/blue-sky.jpg
+categories: events
+authors: ["Rose"]
+---
+<style>
+.twitter-tweet {
+  margin: auto;
+}
+</style>
+
+As every year in spring, last week we organized our third hackathon at ePages.
+This year's hackathon has been all about coding for and further developing our very latest backend architecture ePages BEYOND, adding value to our current core shop software ePages 6 as well as automating internal processes.
```

Comment:
This year's hackathon has been all about trying out new stuff for our very latest backend architecture ePages BEYOND, adding value ePages Now as well as automating internal processes.